tests/serializers/tests.py

```diff
@@ -277,9 +277,7 @@ def test_deferred_field_serialization(self):
         author = Author.objects.defer('name').get(pk=author.pk)
         serial_str = serializers.serialize(self.serializer_name, [author])
         deserial_objs = list(serializers.deserialize(self.serializer_name, serial_str))
-        # Check the class instead of using isinstance() because model instances
-        # with deferred fields (e.g. Author_Deferred_name) will pass isinstance.
-        self.assertEqual(deserial_objs[0].object.__class__, Author)
+        self.assertTrue(isinstance(deserial_objs[0].object.__class__, Author))
```

Comment:
you need to drop the `__class__`, the `object` itself should be an instance of `Author`